### PR TITLE
Add MESMER_M Changelog entries

### PR DESCRIPTION
**Pull request by veni-vidi-vici-dormivi**
These are the Changelog entries for the MESMER-M work of the past few weeks

### CODE REVIEW

**Comment by mathause** [comment]
Thanks for pulling all of that together!

- What do you think about creating a separate section on `Integrating MESMER-M`?
- At some point I started to align the links to the PRs below each other - I found that easier to read and check - up to you
- It's probably easier to compile to docs locally to make sure it's properly formatted

File: CHANGELOG.rst [hunk 1]
```diff
@@ -45,6 +45,44 @@ Internal Changes
...
+- refactoring of MESMER-M functions (
+  `#415 <https://github.com/MESMER-group/mesmer/pull/415>`_,
+  `#434 <https://github.com/MESMER-group/mesmer/pull/434>`_,
+  `#439 <https://github.com/MESMER-group/mesmer/pull/439>`_)
+- add utility functions for MESMER-M (
+  `#418 <https://github.com/MESMER-group/mesmer/pull/418>`_,
+  `#435 <https://github.com/MESMER-group/mesmer/pull/435>`_)
+- adjust the first guess for the power transformer to 0 and 1 (`#429 <https://github.com/MESMER-group/mesmer/pull/429/files>`_)
+- adjust the bounds for the power transformer to all positive numbers for first coefficient (`#446 <https://github.com/MESMER-group/mesmer/pull/446>`_))
+- optimizing (
```
Comment (mathause) [inline CHANGELOG.rst:70]:
```suggestion
- performance optimization (
```

**Review by mathause — COMMENTED** [review]
I quickly read through it and I think it could be a bit more informative for the breaking changes. Maybe you can structure a bit more by power transformer & power transformer as done in https://github.com/MESMER-group/mesmer/blob/main/CHANGELOG.rst#new-features-2. You don't need to separate all of the PRs but maybe the ones that change functionality.

**Comment by mathause** [comment]
I've taken the liberty to restructure the changelog entries - can you take a look? Do you remember the PR where you fixed the month indices? Must have been one of the first ones...

**Comment by veni-vidi-vici-dormivi** [comment]
Yes it was this one: #415, but there was also quite some other stuff going on in that one as well...

File: CHANGELOG.rst [hunk 2]
```diff
@@ -45,6 +46,61 @@ Internal Changes
 - Unshallow the mesmer git repository on rtd (`#456 <https://github.com/MESMER-group/mesmer/pull/456>`_).
   By `Victoria Bauer`_.
 
+Integration of MESMER-M
+^^^^^^^^^^^^^^^^^^^^^^^
+
+This version integrates MESMER-M into the remaining MESMER infrastructure. This includes
+some refactoring, bugfixes and enhancements of the MESMER-M functionality.
+
```
Comment (mathause) [inline CHANGELOG.rst:54]:
```suggestion
some refactoring, bugfixes and enhancements of the MESMER-M functionality. Note
that this led to some numerical changes compared to the MESMER-M publication
(Nath et al., (2022)[https://doi.org/10.5194/esd-13-851-2022]).

```

File: CHANGELOG.rst [hunk 3]
```diff
@@ -45,6 +46,65 @@ Internal Changes
 - Unshallow the mesmer git repository on rtd (`#456 <https://github.com/MESMER-group/mesmer/pull/456>`_).
   By `Victoria Bauer`_.
 
+Integration of MESMER-M
+^^^^^^^^^^^^^^^^^^^^^^^
+
+This version integrates MESMER-M into the remaining MESMER infrastructure. This includes
```
Comment (mathause) [inline CHANGELOG.rst:52]:
```suggestion
This release integrates MESMER-M into the existing MESMER infrastructure. This includes
```

File: CHANGELOG.rst [hunk 4]
```diff
@@ -45,6 +46,65 @@ Internal Changes
...
+some refactoring, bugfixes and enhancements of the MESMER-M functionality. Note
+that this led to some numerical changes compared to the MESMER-M publication
+(Nath et al., (2022)[https://doi.org/10.5194/esd-13-851-2022]).
```
Comment (mathause) [inline CHANGELOG.rst:55]:
```suggestion
(Nath et al., [2022](https://doi.org/10.5194/esd-13-851-2022)).
```

:man_shrugging:

**Review by mathause — APPROVED** [review]

**Pull request merged**